src/azure-cli/azure/cli/command_modules/appservice/utils.py

```diff
@@ -63,3 +63,9 @@ def call(*args, **kwargs):
                 time.sleep(interval_sec)
         return call
     return decorate
+
+
+def _get_resource_group_from_id(arm_id):
```

Comment:
What is the purpose of this helper? is there a reason we need this specifically for static Apps?
@qwordy does CLI have a shared helper for this?

Reply:
from msrestazure.tools import parse_resource_id

Reply:
removed this helper method and used msrestazure.tool